jemalloc-sys/build.rs

```diff
@@ -403,6 +406,10 @@ fn gnu_target(target: &str) -> String {
         "x86_64-pc-windows-msvc" => "x86_64-pc-win32".to_string(),
         "i686-pc-windows-gnu" => "i686-w64-mingw32".to_string(),
         "x86_64-pc-windows-gnu" => "x86_64-w64-mingw32".to_string(),
+        "armv7-linux-androideabi" => "arm-linux-androideabi".to_string(),
+        "aarch64-linux-android" => "aarch64-linux-android".to_string(),
+        "x86_64-linux-android" => "x86_64-linux-android".to_string(),
+        "i686-linux-android" => "i686-linux-android".to_string(),
```

Comment:
In the last three the name does not change so there is no need to add a match arm for them.

Reply:
Removed 😃